src/main/java/tak/Client.java

```diff
@@ -639,10 +639,20 @@ else if(" B".equals(m.group(4)))
 					//Accept a seek
 					else if (game==null && (m = acceptSeekPattern.matcher(temp)).find()) {
 						Seek.seekStuffLock.lock();
+						Log("Entering ACCEPT SEEK client="+ this.getName());
 						try{
 							Seek sk = Seek.seeks.get(Integer.parseInt(m.group(1)));
+							if (sk != null && sk.client == null) { // Tournament Seek
```

Comment:
I wonder if it would be cleaner to have some sort of is_tournament flag as part of creating a new seek

Reply:
Absolutely, I'll leave this thread unresolved until I've added that.